app/scripts/controllers/info.ctrl.js

```diff
@@ -1,8 +1,36 @@
 'use strict';
 
-angular.module('yourStlCourts').controller('InfoCtrl', function ($anchorScroll) {
+angular.module('yourStlCourts').controller('InfoCtrl', function ($anchorScroll,courts, $state,municipalities) {
   var ctrl = this;
+  ctrl.citySearchGroups = [];
+  ctrl.municipalities = municipalities;
+  ctrl.selectedCitySearchGroup = null;
   ctrl.scrollTo = function(id){
     $anchorScroll(id);
   }
+
+  ctrl.onCitySearchGroupSelected  = function(){
+    //$state.go('courtSearchInfo', {courtId : ctrl.selectedCitySearchGroup.court.id});
+  };
+
+  ctrl.groupCourts = function(citySearchGroup) {
+    return citySearchGroup.municipalityCourtCount > 1 ? citySearchGroup.municipalityName : undefined; //undefined causes no group to be made
+  };
+
+  function populateCitySearchGroups() {
```

Comment:
Is this function copied from the homepage? If so, we should prevent the duplication in some way - either by component or factory.

Reply:
This controller should have been deleted